### PR TITLE
[TASK] Update image and screenshot guidelines

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 1]
```diff
@@ -9,167 +9,58 @@
...
-   SVG image with rectangles and numbers
-
-.. figure:: /_Images/edit_me_on_github+shadow.svg
-   :class: with-shadow
-   :alt: Image with arrow
+documentation. These additional graphic elements may be boxes, numbers or arrows.
 
-   SVG Image with arrow
+*   Use sufficient contrast to ensure additional graphic elements are visible
+    across devices and for as many readers as possible, even if the do not see
```
Comment (andreaskienast) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:65]:
```suggestion
    across devices and for as many readers as possible, even if they do not see
```

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 2]
```diff
@@ -9,167 +9,58 @@
...
+    a difference between red and green.
```
Comment (andreaskienast) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:66]: There are more color vision deficiencies than red-green color blindness. Maybe we should make this more clear?

**Review by andreaskienast — CHANGES_REQUESTED** [review]

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 3]
```diff
@@ -9,167 +9,58 @@
 Guidelines for creating images
 ==============================
 
-When embedding images with reST, please add a shadow,
-as described in :ref:`how-to-document-images`.
+Read here how to embed an image into ReST: :ref:`Images <h2document:images>`.
 
-There are various kinds of images used in the documentation. The most
-common are:
+For accessibility reasons always use provide an alt text.
```
Comment (linawolf) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:14]:
```suggestion
For accessibility reasons always provide an alt text.
```

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 4]
```diff
@@ -9,167 +9,58 @@
...
+*   If you need an example site package use :composer:`t3docs/site-package.
+*   If you need example data use :composer:`t3docs/site-package-data`
+*   As personalized usernames are considered best practice always use username
+    "kasper"
+*   Do not install third party extensions unless what you want to demonstrate
+    requires one. If possible use one of the extensions from vendor `typo3` or
+    `t3docs`.
+*   use PNG format (.png file ending)
+*   If you take a screenshot of a full page it should have 1400 x 1050 px
+*   Use only parts of a full page when possible, the flatter the screenshot the
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:51]: shorter from top to bottom then from left to right, how do you call that?

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 5]
```diff
@@ -9,167 +9,58 @@
...
-
-.. figure:: /_Images/edit_me_on_github+shadow.svg
-   :class: with-shadow
-   :alt: Image with arrow
+documentation. These additional graphic elements may be boxes, numbers or arrows.
 
-   SVG Image with arrow
+*   Use sufficient contrast to ensure additional graphic elements are visible
+    across devices and for as many readers as possible, even if they do not see
+    a difference between red and green.
```
Comment (linawolf) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:66]:
```suggestion
    across devices and for as many readers as possible, even if they have
    color vision differences.
```

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 6]
```diff
@@ -9,167 +9,58 @@
...
-   :scale: 70
-
-
-You may however notice, that it might be helpful to point out some relevant part of the
-screenshots. That is what we do by creating images as described in the next section.
-
-
-.. index:: Screenshots; Adding graphics elements
+..  note::
+    We do not have an automatic screenshot tool working with current TYPO3
```
Comment (benjaminkott) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:31]: do we really need to mention this? i would prefer we would only sate the IS state and not mention the was state.

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 7]
```diff
@@ -9,167 +9,58 @@
...
+*   Before adding a screenshot consider if one is necessary. Each new screenshot
+    requires maintenance.
+*   Use a Composer-based installation of the latest LTS release, or dev-main.
+*   Turn the backend into light mode and modern look.
+*   Unless you want to demonstrate features of certain system extensions use
+    a default installation as described in Getting Started:
+    :ref:`Installing TYPO3 with DDEV <t3start:installation-ddev-tutorial>`.
+*   If you need an example site package use :composer:`t3docs/site-package`.
+*   If you need example data use :composer:`t3docs/site-package-data`
+*   As personalized usernames are considered best practice always use username
```
Comment (benjaminkott) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:44]: Can we extend this to First and Last Name`?

Like John/Jane/J. Doe

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst (hunk 6 above)
Comment (benjaminkott) [reply Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:31]: as discussed,. we keep this in as this is a common question that needs to be answered

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 8]
```diff
@@ -9,167 +9,58 @@
...
+    "kasper"
```
Comment (linawolf) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:45]:
```suggestion
    "j.doe"
```

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Basics/GeneralConventions/GuidelinesForImages.rst]: Done

**Review by garvinhicking — APPROVED** [review]

File: Documentation/Basics/GeneralConventions/GuidelinesForImages.rst [hunk 9]
```diff
@@ -9,167 +9,58 @@
...
+    never worked beyond TYPO3 11 LTS.
+
+*   Before adding a screenshot consider if one is necessary. Each new screenshot
+    requires maintenance.
+*   Use a Composer-based installation of the latest LTS release, or dev-main.
+*   Turn the backend into light mode and modern look.
+*   Unless you want to demonstrate features of certain system extensions use
+    a default installation as described in Getting Started:
+    :ref:`Installing TYPO3 with DDEV <t3start:installation-ddev-tutorial>`.
+*   If you need an example site package use :composer:`t3docs/site-package`.
```
Comment (linawolf) [inline Documentation/Basics/GeneralConventions/GuidelinesForImages.rst:42]:
```suggestion
*   If you need an example site package use `t3docs/site-package`.
```

**Pull request merged**